### DISCUSSION TITLE
Merge, what?

### DISCUSSION THREAD

**Discussion by ruguysgoingtrickortreating** [BEEmod/BEE2-items#4182] [Tech Support / Q&A] [answered]
The README file says to merge the package zip into the folder, but I have no clue what it means by that. Sorry for short post I just need help and don't know how to comment on the wordpress thing that led me here.

**Comment by Konclan** [comment, marked as answer]
~Please read the [contributing guidelines](https://github.com/BEEmod/.github/blob/c749cc4941be5f05b0d085a3cb0f973ae131cc49/contributing.md) before posting. The issue tracker isn't for questions or jokes, if you have a question head over to the discord.~

Haha issue conversion go brrr
